mlir/unittests/Dialect/QCO/Utils/test_wireiterator.cpp

```
@@ -0,0 +1,132 @@
+ * SPDX-License-Identifier: MIT
+ *
+ * Licensed under the MIT License
+ */
+
+#include "mlir/Dialect/QCO/Builder/QCOProgramBuilder.h"
+#include "mlir/Dialect/QCO/IR/QCODialect.h"
+#include "mlir/Dialect/QCO/Utils/WireIterator.h"
+
+#include <gtest/gtest.h>
+#include <iterator>
+#include <llvm/Support/Debug.h>
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)